src/member/apps.py

```diff
@@ -3,3 +3,9 @@
 
 class MemberConfig(AppConfig):
     name = 'member'
+
+    def ready(self):
+        # import signal handlers
+        # (every models imported inside handlers will be instantiated as
+        # soon as the registry is fully populated.)
+        import member.signals.handlers  # noqa
```

Comment:
@aaboffill In `apps.py`:

````python
from member.signals.handlers import ensure_pair_remove

...
    def ready(self):
        ClientScheduledStatus = self.get_model('member.ClientScheduledStatus')
        post_delete.connect(ensure_pair_remove, sender=CSS, dispatch_uid="...")
````

And in member.signal.handler:

````python
def ensure_pair_remove(sender, instance, **kwargs):
    if instance.get_pair:
        ClientScheduledStatus.objects.get(pk=instance.get_pair.pk).delete()
````

So split the definition of handler function and the registration of signal handler.